### ISSUE TITLE
Nested groups aren't listed and can't be selected

### ISSUE THREAD

**Issue by whity76** [zammad/zammad#5126]
### Used Zammad Version

6.2.0-1712728678.1fb595f2.jammy and 6.2.0-1712931839.b27a0d24.jammy

### Environment

- Installation method: package
- Operating system: 22.04.4 LTS (Jammy Jellyfish), Azure VM
- Database + version:  10.6.16-MariaDB-0ubuntu0.22.04.1
- Elasticsearch version: 7.10.2, Build: oss/deb/747e1cc71def077253878a59143c1f785afa92b9/2021-01-13T00:42:12.435326Z, JVM: 15.0.1
- Browser + version: Edge for Business 123.0.2420.81

### Actual behaviour

Nested groups are not displayed when creating a new ticket or editing an existing ticket.

Configuration:
![image](https://github.com/zammad/zammad/assets/7700281/946b7410-52c4-458a-b97d-7cb082294190)

New ticket:
![image](https://github.com/zammad/zammad/assets/7700281/6d3e90d8-2af6-41e6-ac63-08553daf2c3b)

Existing ticket:
![image](https://github.com/zammad/zammad/assets/7700281/f30d984c-65d5-4107-9704-db86827bb0e8)

### Expected behaviour

Nestes groups should be display, user should be able to select them.

### Steps to reproduce the behaviour

Just try to open a new ticket or edit an existing one as admin in my installation.

### I'm sure this is a bug and no feature request or a general question.

yes

**Comment by dominikklein** [comment]
Please check if your browser has the correct state of the app (delete the browser cache or check it with a private session).

In general this feature is working and I can not reproduce it.

**Comment by whity76** [comment]
@dominikklein 

Thanks for your answer! I cleaned my browser and even tried 2 new browsers where I never used zammad before, same problem.

After playing around with F12 I stumpled upon:
![image](https://github.com/zammad/zammad/assets/7700281/80d67506-f7d3-4ec8-ae00-6d2ae0e55cc6)

So I deleted all group permissions for my user and added them again. A clean new reload later everything worked and all groups (even the nested ones an a new nested group) are shown in the ticket_create and recent_view forms.
